### DISCUSSION TITLE
Can you tell the difference between upload and download?

### DISCUSSION THREAD

**Discussion by jgelcon** [bpozdena/OneDriveGUI#121] [Q&A]
Hi!
I love the app, I started using it recently on Ubuntu 22.04 and works very good!

I have one question. So far, I couldn't tell if the synchronized files are uploading from local computer, or downloading/downloaded from OneDrive cloud folder. Is there a way to tell the difference on the GUI window?

(By the way, the function to click on the folder containing the synchronized file is very useful)

Thanks a lot, best regards,
Julian

**Comment by bpozdena** [comment]
Hi @jgelcon ,

Thanks.

you should be able to see the current status "downloading/uploading" for each file:
![image](https://github.com/bpozdena/OneDriveGUI/assets/24818591/c05e313b-3010-4c8d-ab4f-4145a448e315)

**Comment by jgelcon** [comment]
Thanks!
Yes, you're right, I can see "Uploading" or "downloading" right in the moment it's syncing. The doubt arises when I am not seeing the GUI at the moment and I only see "Available in [folder]" after sync, as in your screenshot.
In a few cases, I have modified the file in two computers and I want to know I have the most recent version, to go on modifying the current version.
Best,
Julian

**Comment by bpozdena** [comment]
OK, It's something I can probably add together with the timestamps #97 . But I can't provide any ETA at the moment. I just don't have the time lately :-/

**Reply by jgelcon** [reply]
Thanks, it would be great!
I am not very good coding, but if you want to send me simple tasks, maybe I can help somehow.
Best,
Julian
